time.md

```diff
@@ -894,8 +893,8 @@ Here is an example of one of the tests being fixed; try and do the rest yourself
 			t.Errorf("got %q, want %q", gotPrompt, wantPrompt)
 		}
 
-		if game.StartCalledWith != 7 {
-			t.Errorf("wanted Start called with 7 but got %d", game.StartCalledWith)
+		if game.StartedWith != 7 {
```

Comment:
The struct has a StartedWith at this point. Later it also has a StartCalled. Typo, I reckon.